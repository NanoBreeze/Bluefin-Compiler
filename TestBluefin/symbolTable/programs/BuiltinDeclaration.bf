int a;
float b;
bool c;
string d;
void f() {}