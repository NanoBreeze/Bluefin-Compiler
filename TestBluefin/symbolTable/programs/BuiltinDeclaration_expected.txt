declare - a - c_var - t_int
declare - b - c_var - t_float
declare - c - c_var - t_bool
declare - d - c_var - t_string
declare - f - c_function - t_void
enterScope - Level 1
exitScope - Level 1
resolve - int - c_builtinType - t_int
resolve - float - c_builtinType - t_float
resolve - bool - c_builtinType - t_bool
resolve - string - c_builtinType - t_string
resolve - void - c_builtinType - t_void
